### ISSUE TITLE
[REQ] Add a tile showing a thumbnail graph of the in progress training plan

### ISSUE THREAD

**Issue by rjbutler** [cagnulein/qdomyos-zwift#802]
**Is your feature request related to a problem? Please describe.**
It would be nice to know what's coming when using a training plan/ZWO file while in the middle of a workout.  To a lesser extent this would be useful in PowerZone classes (lesser extent because the instructor often tells you and the classes are more uniform)

**Describe the solution you'd like**
Show a thumbnail graph of a training plan or PowerZone class similar to the mini graphs on TrainerDay (see attached image) in a tile during workouts.  Perhaps with a vertical line or pointer showing the current position within the workout

**Describe alternatives you've considered**
Alternatively a preview of the workout or swiping to a screen that shows a screen similar to the end of workout graphs but for an in-progress workout

**Additional context**

![image](https://user-images.githubusercontent.com/505686/167060178-83f4e911-e7ca-493d-ba53-323d0b29c68b.png)

**Comment by cagnulein** [comment]
yes i started this on branch but i would like to do using the web server now
